scripts/verify-support-playbook-break-glass-rehearsal.sh

```
@@ -0,0 +1,166 @@
+    exit 1
+  fi
+}
+
+reject_workstation_paths() {
+  local description="$1"
+  shift
+
+  local macos_home_pattern linux_home_pattern windows_home_pattern workstation_local_path_pattern
+  macos_home_pattern='/'"Users"'/[^[:space:])>]+'
+  linux_home_pattern='/'"home"'/[^[:space:])>]+'
+  windows_home_pattern='[A-Za-z]:\\'"Users"'\\[^[:space:])>]+'
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)